### PR TITLE
Add index on `priority` and `when_added`

### PR DESCRIPTION
**Pull request by bocribbz**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
 envlist =
-	py27-django1.{4,8}
+	py27-django1.{5,8}
 
 [testenv]
 commands = {envpython} run_tests.py
 deps =
 	django_nose
-	django1.4: Django>=1.4, <1.5
+	django1.5: Django>=1.5, <1.6
```
Comment (jlward) [inline tox.ini]: I don't see any reason to support django 1.5 either. Might as well drop everything but 1.8

**Comment by jlward** [comment]
Looks good.

**Comment by jlward** [comment]
Whenever we add this to PolicyStat, we'll either need to re-run migrations for this app or stop handing migrations for this app in PolicyStat.

**Comment by bocribbz** [comment]
> Whenever we add this to PolicyStat, we'll either need to re-run migrations for this app or stop handing migrations for this app in PolicyStat.

I tested locally with the initial migration on our `thirdparty migrations` folder that is already applied, the setting override for `mailer` disabled, and the second migration from this repo, and it was successful.

**Pull request merged**